pkg/tsdb/mssql/mssql.go

```diff
@@ -146,5 +148,14 @@ func (t *mssqlQueryResultTransformer) TransformQueryResult(columnTypes []*sql.Co
 }
 
 func (t *mssqlQueryResultTransformer) TransformQueryError(err error) error {
+	// go-mssql overrides source error why we currently match on string
```

Comment:
The current comment doesn't really parse, does this make sense?
```suggestion
	// go-mssql overrides source error, so we currently match on string
```